spirv_glsl.cpp

```diff
@@ -786,122 +770,36 @@ string CompilerGLSL::layout_for_member(const SPIRType &type, uint32_t index)
 
 const char *CompilerGLSL::format_to_glsl(spv::ImageFormat format)
 {
-	auto check_desktop = [this] {
```

Comment:
enum to string functionality moved to parent class `format_to_string`, retaining only the GLES valid format check